io.enpass.Enpass.yaml

```diff
@@ -0,0 +1,95 @@
+# io.enpass.Enpass.yaml
+app-id: io.enpass.Enpass
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: enpass
+tags: [proprietary]
+separate-locales: false
+finish-args:
+    # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+    # Notification access
+  - --talk-name=org.freedesktop.Notifications
+    # System Tray Icon
+  - --talk-name=com.kde.StatusNotifierWatcher
+    # Secret Service
+  - --own-name=org.freedesktop.secrets
+    # Favicon download
+  - --share=network
+    # Required to save vaults in default location <$HOME/Documents>
+  - --filesystem=xdg-documents
+modules:
+  - name: enpass
+    buildsystem: simple
+    build-options:
+      env:
+        FLATPAK_ID: io.enpass.Enpass
+    build-commands:
+      - |
+
+        mkdir -p ${FLATPAK_DEST}/share/mime/packages
```

Comment:
```suggestion
```

`install` -D does that for you

Reply:
[Resolved](https://github.com/ShobuPrime/flathub/commit/fb8b85a780393498ac9189a026c1d90afa5e450c#diff-4e1d2b12a1a9cb4a6aa05390bf69c7168008557903f720b62b5447c3b1ef7018L32) :heavy_check_mark: